CMakeLists.txt

```diff
@@ -14,8 +14,8 @@ set(CMAKE_ERROR_DEPRECATED FALSE)
 set(CMAKE_WARN_DEPRECATED FALSE)
 
 option(BUILD_TESTS "Enable tests" ON )
-option(BUILD_GO "Enable go wrapper" OFF )
-option(BUILD_PYTHON "Enable python wrapper" OFF)
+option(BUILD_GO "Enable go wrapper" ON )
+option(BUILD_PYTHON "Enable python wrapper" ON)
```

Comment:
Please, do not enable by default. This should be done by Pypi, circle, etc.
Otherwise, this forces a complete build and affects normal C++ development.

Have a look at how setup.py or circle.yml enable specific builds.

Reply:
Circleci and test are work in progress.
Yes missed setting back to OFF. I was just look for travis test. will set  back to OFF